rpc/javascript.go

```diff
@@ -399,6 +399,11 @@ web3._extend({
 			call: 'debug_writeMemProfile',
 			params: 1
 		}),
+		new web3._extend.Method({
+			name: 'replayTransaction',
+			call: 'debug_replayTransaction',
+			params: 4
+		})
```

Comment:
other methods are also not sorted alphabetically (see the stacks and cpuProfile methods), where do you want me to put the function?